README.md

```diff
@@ -1,2 +1,171 @@
+
+##### Available Scripts
+
+* `npm start` - Runs a webpack build with `webpack.config.js` and starts a development server
+* `npm run build` - Runs a webpack build with `webpack.config.js`
+* `npm run release` - Runs an optimized webpack build with `webpack.release.config.js`
+* `npm run serve-release` - Runs an optimized webpack build with `webpack.release.config.js` and starts a development server
+
+##### Configurations
+
+We've included two webpack configuration files in this repository. `webpack.config.js` contains the minimal recommended configuration for getting setup, and configuration for running the development server. `webpack.release.config.js` contains  an optimized configuration for production use.
+
+### Requiring Cesium in your application
+
+Using either the `build` or `release` configurations provided, there are several ways to include Cesium in your application. There is the [CommonJS](http://requirejs.org/docs/commonjs.html) style syntax which uses `require`, and the newer [ES6](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) style syntax which uses the `import` keyword. Both are supported by webpack.
+
+You can also include all modules under the global Cesium object using `Cesium.js', or the individual modules. Requiring individual modules is preferred, resulting in smaller bundle sizes and better performance in your application. See the [Performance Configurations](#performance-configurations) section for more information on improving performance in your application.
+ 
+##### CommonJS Require
+
+ 	var Cesium = require('cesium/Cesium');
+ 	var viewer = new Cesium.Viewer('cesiumContainer');
+
+##### ES6 Style Import
+
+ 	import Cesium from 'cesium/Cesium';
+ 
+##### Require Cesium static asset files
+
+ 	require('cesium/Widgets/widgets.css');
+
+ 	require('cesium/Assets/Textures/pin.svg');
+
+##### Require specific modules from Cesium
+
+	var Color = require('cesium/Core/Color');
+	var c = Color.fromRandom();
+
+or with ES6 Syntax:
+
+	import Color from 'cesium/core/Color';
+	var c = Color.fromRandom();
+
+### Using another Cesium location
+
+We've set the cesium Source location to be what's included with the cesium npm module.
+
+	var cesiumSource = path.resolve(__dirname, 'node_modules/cesium/Source');
+
+However, you may want to use a different version of cesium, like if you've cloned the cesium source code directly. Just set the cesium location to the appropriate path.
+
+	var cesiumSource = path.resolve(__dirname, '../path/to/cesium/Source');
+
+You could even point the cesium to a branch in GitHub or another url in your `package.json` file.
+
+### Source Maps
+
+Enable source maps in development for easier debugging. They are only available when using the development server. There are many [options](https://webpack.js.org/configuration/devtool/). Suggested would be `eval` for fast build and rebuilt time and allowing evaluation of the webpack generated code.
```

Comment:
@mramato Should this read "Suggested would be 'eval' for fast build and rebuild time..."? "rebuild" vs "rebuilt"